core/rawdb/schema.go

```diff
@@ -78,30 +78,30 @@ var (
 )
 
 const (
-	// freezerHeaderTable indicates the name of the freezer header table.
-	freezerHeaderTable = "headers"
+	// FreezerHeaderTable indicates the name of the freezer header table.
+	FreezerHeaderTable = "headers"
```

Comment:
This is coming along! 

Just a thought on exporting these bad boys. In terms of maintaining upstream mergability, I'd _generally_ like to avoid this kind of thing, if possible. 

What if we were to just place an equivalent exported layer on top, maybe something like...

- core/rawdb/schema_remote.go (new file, never has merge conflicts)
```go
const (
  RemoteFreezerHeaderTable = freezerHeaderTable
  RemoteFreezerHashTable = freezerHashTable
  // ...
)
```

With the `Remote` prefix hopefully making it clear that that use case is the reason for it being public.

Reply:
And a git thing, please let's try to break massive renames and other strictly-refactorings into their own commits to make it easier to diff.

Reply:
This was resolved (completed) with https://github.com/etclabscore/core-geth/pull/103/commits/685132e050e946bd0f4fd48f5fb8a75831cc0622